Sofa/Component/SolidMechanics/Spring/src/sofa/component/solidmechanics/spring/StiffSpringForceField.inl

```diff
@@ -43,13 +43,11 @@ StiffSpringForceField<DataTypes>::StiffSpringForceField(SReal ks, SReal kd)
 template<class DataTypes>
 StiffSpringForceField<DataTypes>::StiffSpringForceField(MechanicalState* object1, MechanicalState* object2, SReal ks, SReal kd)
     : SpringForceField<DataTypes>(object1, object2, ks, kd)
-    , d_indices1(initData(&d_indices1, "indices1", "Indices of the source points on the first model"))
-    , d_indices2(initData(&d_indices2, "indices2", "Indices of the fixed points on the second model"))
     , d_lengths(initData(&d_lengths, "lengths", "List of lengths to create the springs. Must have the same than indices1 & indices2, or if only one element, it will be applied to all springs. If empty, 0 will be applied everywhere"))
 {
     this->addAlias(&d_lengths, "length");
 
-    this->addUpdateCallback("updateSprings", { &d_indices1, &d_indices2, &d_lengths, &this->ks, &this->kd}, [this](const core::DataTracker& t)
+    this->addUpdateCallback("updateSprings", { &this->d_springsIndices[0],&this->d_springsIndices[1], &d_lengths, &this->ks, &this->kd}, [this](const core::DataTracker& t)
```

Comment:
I'm not really sure it is the same thing...
In any case you should add alias with deprecated warning message